NewEmergmed/sackett/widgets.py

```diff
@@ -19,10 +22,13 @@ def __init__(self, searchable=False, fill_parent=True, default_text=None, index_
         self.default_text = default_text
         self.index_items = index_items
 
+
+
     def render(self, name, value, attrs=None, choices=(), renderer=None):
         if value is None:
             value = ''
-        final_attrs = self.build_attrs(self.attrs if 'id' not in attrs.keys() else attrs, {'name': name})
+        #final_attrs = self.build_attrs(self.attrs if 'id' not in attrs.keys() else attrs, {'name': name})
```

Comment:
I think your solution is fine so you can delete this commented line; if we need to review it later, we can use the history in git.